### PR TITLE
feat: Push container image to GHCR

### PR DESCRIPTION
**Pull request by onedr0p**
Hi @RemiRigal👋🏼 Awesome project here...

Dockerhub has aggressive pull rate limits, it would be awesome if you could push the image to Github Container Registry (GHCR) too.

Dockerhub was one of the best years ago... GHCR is completely free for open source projects. It's also nice in that it integrates into your repo under the packages section. Take a look at my project [here](https://github.com/onedr0p/exportarr) and look at the packages on the right side.

Thanks!

Edit: This also bumps deps in the workflow

### CODE REVIEW

File: .github/workflows/dockerhub_build_push.yml [hunk 1]
```diff
@@ -9,29 +9,37 @@ on:
...
+          password: ${{ secrets.GITHUB_TOKEN }}
       - name: Extract metadata (tags, labels) for Docker
         id: meta
-        uses: docker/metadata-action@v3
+        uses: docker/metadata-action@v5
         with:
-          images: remirigal/plex-auto-languages
+          images: |
+            docker.io/remirigal/plex-auto-languages
+            ghcr.io/${{ github.repository }}
```
Comment (onedr0p) [inline .github/workflows/dockerhub_build_push.yml:37]:
```suggestion
            ghcr.io/remirigal/plex-auto-languages
```
This needs to be changed to be lower case

**Comment by onedr0p** [comment]
@RemiRigal after this is merged and a release it cut you may have to make the package public. Thanks 👍🏼

File: .github/workflows/dockerhub_build_push.yml [hunk 2]
```diff
@@ -9,29 +9,40 @@ on:
...
+        uses: docker/login-action@v3
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```
Comment (onedr0p) [inline .github/workflows/dockerhub_build_push.yml:33]: This secret is auto-generated in the pipeline, no need to create a PAT.

**Comment by onedr0p** [comment]
This project appears to be dead, closing.

**Comment by RemiRigal** [comment]
Hi @onedr0p, apologies for the late reply I was busy in the past few days. This project is far from dead, I don't have as much time as I'd like to work on it but it remains functional and supported.

Thanks for the PR.

**Pull request merged**